pkgs/data/themes/everforest-gtk-theme/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  pname = "Everforest-GTK-Theme";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://github.com/Fausto-Korpsvart/Everforest-GTK-Theme/archive/refs/heads/master.tar.gz";
+    sha256 = "2gmSMW/yUdinvZ/Jaw6GyfEL/CBqYKtw0QrYWy9KrL0=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/themes
+    rm -rf README.md LICENSE icons extra
+    cp -r themes/* $out/share/themes/
+  '';
+
+  meta = with lib; {
+    description = "Everforest theme for GTK based desktop environments";
+    homepage = "https://github.com/Fausto-Korpsvart/Everforest-GTK-Theme";
+    license = licenses.gpl3Only;
```

```suggestion
    license = licenses.gpl3Plus;
```

https://github.com/Fausto-Korpsvart/Everforest-GTK-Theme/blob/8481714cf9ed5148694f1916ceba8fe21e14937b/themes/Everforest-Dark-B/metacity-1/metacity-theme-3.xml#L6